unity-client/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -1117,5 +1117,7 @@ public static void RequestUserProfile(string userId)
             stringPayload.value = userId;
             SendMessage("RequestUserProfile", stringPayload);
         }
+
+        public static void ReportAvatarFatalError() { SendMessage("ReportAvatarFatalError"); }
```

Comment:
Should we start moving this messages to a constants? So we can see all types of messages that we have quickly

Reply:
In this specific case I think to create constants for each kernel function name is not very important as each name is used only once, but on the other hand I also think that it would visually improve the code. Anyway, if you don't mind, I prefer to separate this kind of changes from this PR.